app/services/plans/update_service.rb

```diff
@@ -97,6 +98,24 @@ def charge_model(params)
       model
     end
 
+    def process_minimum_commitment(plan, params)
+      minimum_commitment = plan.minimum_commitment || Commitment.new(plan:, commitment_type: 'minimum_commitment')
```

Comment:
As I can see this constant is used multiple times. Can we maybe use this one based on enum value, or create a constant for it?